apps/src/labs/labRedux.ts

```diff
@@ -0,0 +1,39 @@
+// This is not yet a complete store for lab-related state, but it can
+// be used to hold lab-specific information that is not available in any
+// other existing redux store.  It also holds useful state for newer labs
+// that use LabContainer.
+
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+
+interface LabState {
+  isLoading: boolean;
+  isPageError: boolean;
+  currentLevelId: string | undefined;
```

Comment:
I might have missed this in a separate discussion - but why do we need to track currentLevelId here when it's already available in a different redux store?

Reply:
Ah, this was because `progressRedux` had its `currentLevelId` as `undefined` when in a level page.  I was wary about changing that behaviour, but after our offline discussion, I'm going to try using it for that purpose.